### PR TITLE
Feature: Create PSAT landing page and dashboard

### LINKED ISSUE #799: Redesign PSAT Landing page
## Feature Request

**Description:**
Design PSAT Landing page to provide easy access to all features as a tool dashboard

### PR DESCRIPTION
**Pull request by mohdsayed**
## Description

Redesign the PSAT landing page and add a dashboard with pinned items and features for easy navigation.

## Relevant Technical Choices

- Remove the iframe and add static content.
- Map internal links for each item in the dashboard.
- Add internal link in PSAT landing page for dashboard.
- Display the landing page when users first install the PSAT extension. On subsequent visits, show the dashboard.
- Add top-level icons in collapsed sidebar.

## Testing Instructions

- Remove and install PSAT extension, you should be able to see the landing page.
- Open extension in a tab, you should now see dashboard.
- All links and buttons in the dashboard should navigate to the related pages.
- Top level icons should be visible in the collapsed sidebar.

## Screenshot/Screencast

Landing Page
image

Dashboard
image

---

## Checklist

- [x] I have thoroughly tested this code to the best of my abilities.
- [x] I have reviewed the code myself before requesting a review.
- ~[ ] This code is covered by unit tests to verify that it works as intended.~ NA
- [x] The QA of this PR is done by a member of the QA team (to be checked by QA).

Fixes #799

### CODE REVIEW

**Comment by mohdsayed** [comment]
@pavanpatil1 Ready for QA

**Comment by pavanpatil1** [comment]
- [ ] The button design should be consistent throughout.

Screenshot 2024-09-06 at 11 30 26 AM

---

- [ ] On the landing page, if CDP is enabled, the reload notice does not display correctly.

Screenshot 2024-09-06 at 11 46 52 AM

---


- [ ] Could we move the dashboard section above the cookie section? The dashboard typically appears first, and from there, we can navigate to the other sections below.

Screenshot 2024-09-06 at 12 18 04 PM

---

- [ ] The color code on the dashboard does not match the one on the "Advertising" landing page. The landing page uses rgb(37, 99, 235), while the dashboard page uses rgb(26, 115, 232 )



Screenshot 2024-09-06 at 12 35 25 PM

---

**Comment by milindmore22** [comment]
- [ ] Disappearing logos 
The Privacy Sandbox logos on the PSAT landing page disappear when you toggle open the sidebar menus.

https://github.com/user-attachments/assets/8e4549a4-61f6-48d6-96de-0d03f9c84140

**Comment by mohdsayed** [comment]
@pavanpatil1 

> The button design should be consistent throughout.

It's the same button used in dashboard. We wouldn't want to use the blue button.

>  On the landing page, if CDP is enabled, the reload notice does not display correctly.

Goo catch! Fixed

>  The logo currently appears in both the header and navigation sections. Could we streamline this by showing the logo in the header only? Also, the logo becomes invisible when navigating to the private audience or site boundary pages.

That makes sense, updated to use the same design from landing pages.

>  Could we move the dashboard section above the cookie section? The dashboard typically appears first, and from there, we can navigate to the other sections below.

I had initially kept it in the beginning; however, it was later decided that we wanted to keep it after “Privacy Sandbox”.

>  The color code on the dashboard does not match the one on the "Private Advertising" landing page. The landing page uses rgb(37, 99, 235), while the dashboard page uses rgb(26, 115, 232 )

Fixed

> The 'Private Advertising' logo differs between the landing page and the dashboard.

Fixed


@milindmore22 

>  Disappearing logos
The Privacy Sandbox logos on the PSAT landing page disappear when you toggle open the sidebar menus.

Fixed.

**Comment by milindmore22** [comment]
- [ ] The border for dashboard boxes is different from other box borders.

image
image

**Comment by fellyph** [comment]
When the user clicks on the cookie table on the dashboard before running an audit, the link goes to the cookie landing page. Should we redirect the user to the cookie table after the user presses "analyses page"?
 

https://github.com/user-attachments/assets/c68144fd-7b21-4179-b4fb-c3d8b2de4784

File: packages/extension/src/view/devtools/components/dashboard/constants.ts [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+} from '@google-psat/design-system';
+
+export const PINNED_ITEMS = [
+  {
+    name: 'Protected Audience',
+    icon: GroupsIcon,
+    sidebarKey: SIDEBAR_ITEMS_KEYS.PROTECTED_AUDIENCE,
+  },
+  {
+    name: 'Cookies',
```
Comment (mayan-000) [inline packages/extension/src/view/devtools/components/dashboard/constants.ts:36]: The translation is available for most names or descriptions, I18n can be used.

File: packages/extension/src/view/devtools/components/dashboard/constants.ts [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+      heading: 'text-blue-600',
+    },
+    buttons: [
+      {
+        name: 'Topics',
+        sidebarKey: SIDEBAR_ITEMS_KEYS.TOPICS,
+      },
+      {
+        name: 'Attribution',
+        sidebarKey: SIDEBAR_ITEMS_KEYS.ATTRIBUTION,
```
Comment (mayan-000) [inline packages/extension/src/view/devtools/components/dashboard/constants.ts:81]: Wrong key, it's ATTRIBUTION_REPORTING

**Comment by mohdsayed** [comment]
@fellyph 
> The dashboard should adapt to small screens and break the layout into two and one-column layout

Fixed

> When the user clicks on the cookie table on the dashboard before running an audit, the link goes to the cookie landing page. Should we redirect the user to the cookie table after the user presses "analyses page"?

I think instead of that, if the table is not available, that should not be displayed, pushed an update for it.

**Review by mayan-000 — APPROVED** [review]

**Pull request merged**